magellan-library/src/main/java/com/wealthfront/magellan/init/Magellan.kt

```diff
@@ -14,23 +15,35 @@ public object Magellan {
   internal var logDebugInfo: Boolean = false
   internal var disableAnimations: Boolean = false
   internal var customDefaultTransition: MagellanTransition = DefaultTransition()
-  public var navigationRequestHandler: NavigationRequestHandler? = null
+  public var navigationOverrides: Set<NavigationOverride> = emptySet()
```

Comment:
hmm this should be a `list` instead of a `set`. If there are multiple overrides desired, we want to give developers an opportunity to order them